config.json

```diff
@@ -159,7 +159,7 @@
         "name": "Resistor Color",
         "uuid": "53be6837-c224-45f1-bff3-d7f74d6285ce",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["arrays"],
```

Comment:
Maybe better: arrays and array-analysis ?

Reply:
Should we just put array-analysis then as arrays is a required concept for array-analysis ?

Reply:
eh. So we MUST list _direct_ dependencies, we CAN list more. Arrays makes sense (intuitively. and how the storage is done) and I like adding analysis because that's what you'll use to find the answer. I *will* approve just `array-analysis`. Do whatever you think is right.